en/docs/security-announcements/security-advisories/2026/WSO2-2026-4608.md

```
@@ -0,0 +1,126 @@
+---
+title: WSO2-2026-4608
+---
+
+# WSO2-2026-4608
+
+Content to be added.
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)